src/gobexport/export.py

```diff
@@ -163,6 +168,22 @@ def _export_collection(host, catalogue, collection, product_name, destination):
     logger.info("Export completed")
 
 
+def cleanup_datefiles(connection, container, filename):
+    key = re.sub(r"\d{8}", "\\\d{8}", filename)  # Detect dates and replace the date by it's regex
+    if key == filename:
+        # No dates in filename, nothing to do
+        return
+
+    logger.info(f'Clean previous files for {filename}.')
+    try:
+        for item in get_full_container_list(connection, container):
+            if re.match(key, item['name']) and item['name'] != filename:
+                delete_object(connection, container, item)
+                logger.info(f'File {item["name"]} deleted.')
+    except Exception:
+        pass
```

Comment:
Why hide all exceptions? I think it will be better to see if something went wrong.